src/components/PeopleTable.tsx

```diff
@@ -10,635 +53,80 @@
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Name
-              <a href="#/people?sort=name">
+              <NavLink
+                to={`${pathname}?${searchParams.toString()}`}
+                onClick={event => {
+                  event.preventDefault();
+                  handleSort('name');
+                }}
+              >
                 <span className="icon">
-                  <i className="fas fa-sort" />
+                  <i className={`fas ${iconClass('name', sort, order)}`} />
                 </span>
-              </a>
+              </NavLink>
             </span>
           </th>
-
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Sex
-              <a href="#/people?sort=sex">
+              <NavLink
+                to={`#${pathname}?${searchParams.toString()}`}
+                onClick={event => {
+                  event.preventDefault();
+                  handleSort('sex');
+                }}
+              >
                 <span className="icon">
-                  <i className="fas fa-sort" />
+                  <i className={`fas ${iconClass('sex', sort, order)}`} />
                 </span>
-              </a>
+              </NavLink>
             </span>
           </th>
-
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Born
-              <a href="#/people?sort=born&amp;order=desc">
+              <NavLink
+                to={`#${pathname}?${searchParams.toString()}`}
+                onClick={event => {
+                  event.preventDefault();
+                  handleSort('born');
+                }}
+              >
                 <span className="icon">
-                  <i className="fas fa-sort-up" />
+                  <i className={`fas ${iconClass('born', sort, order)}`} />
                 </span>
-              </a>
+              </NavLink>
             </span>
           </th>
-
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Died
-              <a href="#/people?sort=died">
+              <NavLink
+                to={`#${pathname}?${searchParams.toString()}`}
```

Comment:
The `to` attribute in `NavLink` should not include a hash (`#`) unless it is intended for anchor links. Consider removing the `#` to ensure proper navigation.